cli/cmd/plugin/diagnostics/Makefile

```diff
@@ -21,12 +21,11 @@ test: ## Run unit testing suite
 
 .PHONY: test-pr-check
 test-pr-check: ## Run e2e testing suite for PR check
```

Comment:
Do we need this target? Should probably consolidate down to just the one expected target of `e2e-test`.

Reply:
+1